agent/http-server.c

```diff
@@ -104,6 +106,29 @@ check_file_path (HwangsaeHttpServer * server, gchar * edge_id, gchar * file_id)
   return NULL;
 }
 
+gchar *
+hwangsae_http_server_get_url (HwangsaeHttpServer * server, gchar * edge_id,
+    gchar * file_id)
+{
+  g_autofree gchar *file_path = NULL;
+  g_autofree gchar *local_ip = NULL;
+  gchar *url;
```

Comment:
Nitpick: this variable is useless. In the both instances where you're doing `return url;` you can simply return the value without assigning it to a variable, like `return g_strdup("");`.

Reply:
Yes, I just thought it was more readable in this way, to make it clear that in this case URL == "", again, no problem in change it.